website/docs/components/toast/partials/guidelines/guidelines.md

```diff
@@ -107,31 +64,41 @@ All Toasts have icons by default that are intentionally tied to the Toast color.
 
 Icons within `neutral` and `highlight` Toasts can be replaced with other icons. Change them only when the new icon provides the user with extra value; otherwise, use the default icon provided.
 
-<Doc::Layout @spacing="12px" @direction="vertical">
-  <Hds::Toast @color="neutral" @icon="running" @onDismiss={{this.noop}} as |T|>
-    <T.Title>Plan running</T.Title>
-    <T.Button @text="Button" @color="secondary" />
-    <T.LinkStandalone @color="secondary" @icon="plus" @iconPosition="leading" @text="Link text" @href="#" />
-  </Hds::Toast>
-  <Hds::Toast @color="success" @icon="check-circle" @onDismiss={{this.noop}} as |T|>
-    <T.Title>Plan finished</T.Title>
-    <T.Button @text="Button" @color="secondary" />
-    <T.LinkStandalone @color="secondary" @icon="plus" @iconPosition="leading" @text="Link text" @href="#" />
-  </Hds::Toast>
-</Doc::Layout>
+![Two toasts, one neutral with a non-standard icon and one success with a standard icon.](/assets/components/toast/toast-icon-neutral-and-success.png)
 
 ## Size
 
-Toasts can be sized between 360px and 500px wide. Anything wider than 500px will lead to implementation issues and take up too much of the screen space. 
+Toasts can be sized between 360px and 500px wide. Anything wider than 500px is considered disruptive to the user experience and may cover important information on the page.
 
 ## Placement
 
-Toasts should appear in the bottom right corner of the screen with a margin of 32px from the bottom and a 24px from the right side of the viewport. 
+Toasts appear in the bottom right corner of the viewport with a margin of 32px from the bottom and 24px from the right side of the viewport. If a warning or error is contextual to a specific UI feature, such as a form, directly associate an Alert to that feature instead of a Toast.
 
-![Toast placement example](/assets/components/toast/toast-placement.png =600x*)
+![](/assets/components/toast/toast-placement.png =600x*)
 
 ### Displaying multiple Toasts
 
-When displaying multiple Toasts, they should stack vertically with a 16px margin between each Toast.
+When displaying multiple Toasts, they should stack vertically with a 16px margin between each Toast. For consistency, each stacked Toast should be the same width.
+
+![](/assets/components/toast/toast-placement-multiple.png =600x*)
+
+## When to persist or timeout
+
+A Toast can either persist until the user dismisses it or be automatically dismissed via a timeout. 
+
+If a Toast contains actions or critical information (often associated with a warning or critical toast), we recommend that it persist until the user dismisses it. Keep in mind that when a Toast persists, it will remain visible across pages until dismissed.
```

Comment:
Do we typically avoid saying "we"? Maybe this should be 
```suggestion
If a Toast contains actions or critical information (often associated with a warning or critical toast), it should persist until the user dismisses it. Keep in mind that when a Toast persists, it will remain visible across all pages until dismissed.
```

Reply:
[We do not](https://docs.google.com/document/d/1WyoJVpWFVgWbCnZ28WW0gRJwJlj-qV5oY0ExhDR2obs/edit#heading=h.p3yb1ajhzcep). We also often lean into our principle of "[guidance over control](https://helios.hashicorp.design/about/principles)" which is why you'll see us use "recommend" a lot. 

That said, this should probably be more prescriptive since it's a major usability concern. I also like that your version is more concise and flows more naturally.